src/util/api/formatResponseObject.ts

```diff
@@ -0,0 +1,65 @@
+import { camelCase, mapKeys, pick, rearg } from "lodash";
```

Comment:
About the location of this helper.

We also have an API folder in `src` root that has an util file. I don't know what we would do to improve it, but we should somehow clear up the responsibilities between `/ util/api` from `/api/utils`. There are multiple levels of conflict.

Perhaps the should rename `src/api` to something else , like `src/client` 🤔

Reply:
Totally forgot that there was an api folder under src. I'll rename that one to `client` as you suggested.